### ISSUE TITLE
Abnormally small slice capacity

### ISSUE THREAD

**Issue by danluu** [BitFunnel/BitFunnel#320]
When we ingest the `chunked1` corpus with our setup we get a sliceCapacity of 64. In addition to contributing to #319, we should expect this to cause some unusual performance artifacts.

I'm filing this as separate from #319 since we could conceivably decide to fix that so that it works even when we have unusually small slices.

**Comment by danluu** [comment]
I don't think I can give this the attention necessary to debug right now, but if we increase the block size BitFunnel crashes when running iteration 65. Maybe we have another hardcoded limitation that we're hitting?

It segfaults (bad memory access) on

~~~
        if (m_diagnosticStream != nullptr && 
            m_diagnosticStream->IsEnabled("bytecode/opcode"))
~~~

**Comment by danluu** [comment]
After looking at this in the light of day, I think the above fix corrupts the intent of the dedupe buffer. I may temporarily disable this until I figure it out.

**Comment by danluu** [comment]
This was blocked by a bug in the dedupe buffer, but that was fixed by ef04f39d388bb72dd435f8b50d57f9784c3b1ecb and 483d1e7f49f45ee4a705034b6613d96e1cf9c0ee. This is now also "fixed" by hardcoding a larger constant. We should have something better, but this works for now.
